app/views/news/index.html.erb

```diff
@@ -16,7 +16,7 @@
 
         <div class='g articles__content'>
           <div class='gi articles__body'>
-            <% #= markdown article.body[0..200] %>
+            <p><%= article.summary_text %></p>
```

Comment:
i think actual truncate call should be here, and the other method just call like "plain_text" or something